src/media/css/site.styl

```diff
@@ -284,7 +284,11 @@ a:active {
     }
     section.full > div {
         padding: 30px 0;
-        width: $max-site-width;
+        max-width: $max-site-width;
+    }
+    .placeholder + .placeholder section.full,
```

Comment:
Should we apply this to all section.full?

![screen shot 2015-02-17 at 7 35 39 pm](https://cloud.githubusercontent.com/assets/674727/6242047/457ce8ae-b6dc-11e4-8357-6ce91e1a078a.png)

Should we apply this to all section.full?

![screen shot 2015-02-17 at 7 35 39 pm](https://cloud.githubusercontent.com/assets/674727/6242047/457ce8ae-b6dc-11e4-8357-6ce91e1a078a.png)